Makefile

```diff
@@ -92,6 +92,9 @@ build: fmt vet
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run cmd/operator/main.go
 
+daemon: fmt vet
+	go run cmd/daemon/main.go
+
```

Comment:
WICD is meant to be a windows exe, instead of having the endpoint use `go run`, please make this compile an executable called `windows-instance-config-daemon.exe`, you will need to cross compile for Windows by setting `GOOS`. the exe should be saved to the same directory that the WMCO binary is outputted to.

Reply:
You can use how we [build](https://github.com/openshift/windows-machine-config-bootstrapper/blob/38466b767564bddeeb68a6a9fab708dd73a937ce/Makefile#L8) the wmcb.exe as a reference.

Reply:
Thanks! Fixed